artview/components/plot_radar.py

```diff
@@ -104,14 +114,18 @@ def __init__(self, Vradar, Vfield, Vtilt, Vlims=None,
 
         # Find the PyArt colormap names
 #        self.cm_names = [m for m in cm.datad if not m.endswith("_r")]
-        self.cm_names = ["pyart_" + m for m in pyart.graph.cm.datad if not m.endswith("_r")]
+        self.cm_names = ["pyart_" + m for m in pyart.graph.cm.datad
+                         if not m.endswith("_r")]
         self.cm_names.sort()
 
         # Create tool dictionary
         self.tools = {}
 
-        # Set up Default limits
-        self._set_default_limits()
+        # Set up Default limits and cmap
+        if Vlims is None:
+            self._set_default_limits(strong=False)
+        if Vcmap is None:
+            self._set_default_cmap(strong=False)
 
```

Comment:
Aha, now I think I am seeing the versatility of the shared limits and cmap variable.